include/util.h

```diff
@@ -34,4 +34,9 @@ const char *sway_wl_output_subpixel_to_string(enum wl_output_subpixel subpixel);
 
 bool set_cloexec(int fd, bool cloexec);
 
+#undef MAX
+#define MAX(a, b)  (((a) > (b)) ? (a) : (b))
```

Comment:
It looks like `fmin` and `fmax` are used throughout the code base so let's just use those instead

Reply:
For some parts that can work but not all code is floating point where `fmin`/`fmax` applies.

Reply:
The magic of automatic number type conversions actually makes the straight swap work fine. I don't really see a situation where this can break although I generally distrust these automatic conversions.